mscore/shortcut.cpp

```diff
@@ -2826,7 +2826,7 @@ Shortcut Shortcut::_sc[] = {
          MsWidget::MAIN_WINDOW,
          STATE_NORMAL | STATE_NOTE_ENTRY | STATE_PLAY,
          "parts",
-         QT_TRANSLATE_NOOP("action","Parts…"),
+         QT_TRANSLATE_NOOP("action","P&arts…"),
```

Comment:
Does this really not conflict with the "A" for Save As?  Next in line would be "R" I guess.

BTW, we've had issues in the past with using mnemonics A-G, they would get misinterpreted as note input commands.  I *think* this is solved, as "F" works for Frames in the Add menu, at least on my system.  But, something to keep in mind and test.

Reply:
Ooops, R is taken too, I guess "T" then.

Reply:
Yup, missed that tpo

Reply:
Fixed. Meanwhile you can have those all in 3.5.x via the en_US, en_GB and de translations